meinberlin/apps/kiezradar/models.py

```diff
@@ -182,27 +182,44 @@ def get_search_profiles_for_project(project: Project) -> QuerySet[SearchProfile]
         & (Q(status__status=status) | Q(status__isnull=True))
         & Q(plans_only=False)
         & (Q(organisations__in=[project.organisation]) | Q(organisations__isnull=True))
-        & (Q(districts__in=[project.administrative_district]) | Q(districts__isnull=True))
+        & (
+            Q(districts__in=[project.administrative_district])
+            | Q(districts__isnull=True)
+        )
         & (
             Q(project_types__participation=ProjectType.PARTICIPATION_CONSULTATION)
             | Q(project_types__isnull=True)
         )
         & Q(disabled=False)
     )
-    search_term = project.name
+    search_term = (
+        project.name + " " + project.description + " " + project.organisation.name
+    )
+    if project.administrative_district:
+        search_term += " " + project.administrative_district.name
+    for topic in project.topic_names:
+        search_term += " " + topic
     if connection.vendor == "postgresql":
         # django has some postgresql-only search tools which are much better
-        query = "|".join(search_term.split(" "))
-        search_query = SearchQuery(query, search_type="raw")
-        search_profiles = search_profiles.annotate(search=SearchVector("query__text"))
+        query = "|".join(search_term.split())
+        # TODO: if we ever support more languages than german, the config would either have to depend on the user
+        #  language or use a library like nltk to sanitize multiple languages.
+        search_query = SearchQuery(query, search_type="raw", config="german")
```

Comment:
@m4ra I set the language to german now which seems to work. Also added a note on more languages